charts/datahub/subcharts/datahub-frontend/templates/deployment.yaml

```diff
@@ -191,8 +191,10 @@ spec:
             {{- else }}
               value: {{ .clientSecret }}
             {{- end }}
+            {{- if $.Values.ingress.enabled }}
```

Comment:
I understand the change and I agree with it (not that you should care bc I am not a maintainer)
But this is adding a breaking change and should bring documentation alongside it. People need to know that they need to input this env var someway else.

Reply:
Yes, people are actively using the workaround that having a disabled ingress does not actually disable the entire section of values. The host variable can be set regardless of whether you are using the ingress or not. I think we would rather see this bubbled up to a defined value in values.yaml with failover to the current value if not set rather than encouraging people to utilize extraEnvs which are really more intended as a way to allow for customizations rather than base functionality.